components/server/ee/src/billing/billing-mode.ts

```diff
@@ -15,12 +15,12 @@ import { StripeService } from "../user/stripe-service";
 import { BillingMode } from "@gitpod/gitpod-protocol/lib/billing-mode";
 import { TeamDB, TeamSubscription2DB, UserDB } from "@gitpod/gitpod-db/lib";
 import { Plans } from "@gitpod/gitpod-protocol/lib/plans";
-import { AttributionId } from "@gitpod/gitpod-protocol/lib/attribution";
+import { AttributionId, TeamAttributionId, UserAttributionId } from "@gitpod/gitpod-protocol/lib/attribution";
 import { TeamSubscription2 } from "@gitpod/gitpod-protocol/lib/team-subscription-protocol";
 
 export const BillingModes = Symbol("BillingModes");
 export interface BillingModes {
-    getBillingMode(attributionId: AttributionId, now: Date): Promise<BillingMode>;
+    getBillingMode(attributionId: UserAttributionId | TeamAttributionId, now: Date): Promise<BillingMode>;
```

Comment:
nit: I don't think this is needed as `AttributionId = UserAttributionId | TeamAttributionId` already.

Reply:
@AlexTugarev Thanks for the catch! This was part of a debugging that accidentally made its way into the commit 🙈 Good thing this is still a draft